test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -1,5 +1,7 @@
 using namespace System.Diagnostics
 
+Import-Module $PSScriptRoot\..\Common\Test.Helpers.psm1
```

Comment:
In the PR I shall move `Test.Helpers.psm1` to `test\Tools\Modules` folder and modify `Start-PSPester` to load the module.

To change the whole structure we need to open new Issue. @lzybkr Could you move your proposal in this new Issue?